packages/plugin-core/src/commands/CapitalizeSelectionCommand.ts

```diff
@@ -0,0 +1,35 @@
+import { DENDRON_COMMANDS } from "../constants";
+import { VSCodeUtils } from "../utils";
+import { BasicCommand } from "./base";
+
+type CommandOpts = {};
+
+type CommandInput = {};
+
+type CommandOutput = void;
+
+export class CapitalizeSelectionCommand extends BasicCommand<
+  CommandOpts,
+  CommandOutput
+> {
+  static key = DENDRON_COMMANDS.CAPITALIZE_SELECTION.key;
+  async gatherInputs(): Promise<CommandInput | undefined> {
+    return {};
+  }
+  async execute() {
+    const maybeTextEditor = VSCodeUtils.getActiveTextEditor();
+
+    if (maybeTextEditor) {
+      const text = maybeTextEditor.document.getText(maybeTextEditor.selection);
+
+      const transformedText = text
+        .split(" ")
+        .map((token) => `${token[0].toUpperCase()}${token.slice(1)}`)
```

Comment:
What would happen for unicode input? eg. 🌱

Reply:
It's generally unicode safe, i.e.:

```
> '👍'.toUpperCase();
'👍'
```

It does look like there are [one or two very minor edge cases](https://stackoverflow.com/a/30763402/2129808) when dealing with other languages, so I changed it to toLocaleUpperCase() instead to be on the safe side: 2f3929be